tests/ea/resume.cpp

```diff
@@ -120,17 +120,20 @@ BOOST_AUTO_TEST_CASE(test_nsga2_resume) {
   typedef ea::Nsga2<phen_t, eval_t, stat_t, modifier_t, Params> ea_t;
   ea_t ea;
   ea.run();// 101 generations
+  std::cout<<"nsga2 done, running some tests..."<<std::endl;
```

Comment:
Why not? We need to be able to diagnose why the unit test is not working and I do not think Travis allows us to write what we need to a file. Similarly, we need to be able to ask the full output for bug reports.

Besides, what is the downside? A verbose output? Who cares of verbose tests?